cmake/modules/FindEmbree.cmake

```diff
@@ -0,0 +1,64 @@
+
+#
+# This source file is part of appleseed.
+# Visit https://appleseedhq.net/ for additional information and resources.
+#
+# This software is released under the MIT license.
+#
+# Copyright (c) 2018 Fedor Matantsev, The appleseedhq Organization
```

Comment:
Did you actually write this file? Doesn't Embree provide one?

Reply:
This one was authored by me. Embree doesn't provide one.